lsp/tsconfig.json

```diff
@@ -12,6 +12,12 @@
     },
     "files": [],
     "references": [
+        {
+            "path": "./app/aws-lsp-buildspec-binary"
```

Comment:
I see we have separate binaries for buildspec and other json/yaml-based DSLs. Did we not plan to have a single binary that can serve different kinds of json/yaml-based DSLs? At least until we run into limitations thereof.

Reply:
We have an investigation spike planned to assess the impacts of running multiple language servers, so for now I've created two separate binaries (one for each of buildspec and cloudformation, which will be in a follow-up PR) which will provide the means for experimenting